project_files/bank_stripped.js

```diff
@@ -0,0 +1,37 @@
+class Printer {     // Responsibility - Print account history
```

Comment:
Conventionally, use directories to organise your files

so a `src` directory to hold source code
+ each class should have its own file named after it